package.json

```diff
@@ -36,9 +36,9 @@
     "node": ">=7.6"
   },
   "scripts": {
-    "test": "npm run lint && NODE_ENV=test mocha --require should --reporter spec test/*.test.js",
-    "test-cov": "NODE_ENV=test node ./node_modules/.bin/istanbul cover ./node_modules/.bin/_mocha -- --require should test/*.test.js",
-    "test-travis": "npm run lint && NODE_ENV=test node ./node_modules/.bin/istanbul cover ./node_modules/.bin/_mocha --report lcovonly -- --require should test/*.test.js",
+    "test": "npm run lint && NODE_ENV=test mocha --exit --require should --reporter spec test/*.test.js",
+    "test-cov": "NODE_ENV=test node ./node_modules/.bin/istanbul cover ./node_modules/.bin/_mocha -- --exit --require should test/*.test.js",
```

Comment:
`--exit` is needed due to [this change in mocha](https://boneskull.com/mocha-v4-nears-release/#mochawontforceexit)